src/pyhf/pdf.py

```diff
@@ -549,7 +499,7 @@ def _create_nominal_and_modifiers(self, config, spec):
         return mega_mods, _nominal_rates
 
     def expected_auxdata(self, pars):
-        return self.constraint_model.expected_data(pars)
+        return self.make_pdf(pars).pdfobjs[1].expected_data()
```

Comment:
why did this get changed? `self.constraint_model` still works here, or have I missed something...

Reply:
I removed logpdf from self.constraint_model